common/shlibs

```diff
@@ -1686,7 +1686,7 @@ libIlmImfUtil-2_4.so.24 libopenexr-2.4.0_1
 libGraphicsMagick.so.3 libgraphicsmagick-1.3.19_1
 libGraphicsMagick++.so.12 libgraphicsmagick-1.3.22_1
 libGraphicsMagickWand.so.2 libgraphicsmagick-1.3.19_1
-liblensfun.so.1 lensfun-0.3.2_1
+liblensfun.so.1 lensfun-0.3.3_1
```

Comment:
Don't do this if the soname doesn't change

Reply:
How do you tell if it changed? It seems like it changed?

Reply:
The soname is in the first column and it didn't change. Second column is the package with the minimal version which provides this soname.

Reply:
xbps-src would exit with an error if soname changed and you didn't change it in common/shlibs.